src/Mod/Material/Gui/CMakeLists.txt

```diff
@@ -154,11 +154,7 @@ SET(MatGuiImages
     Resources/images/default_image.png
 )
 
-SET(Material_Ui_Files
-    Resources/ui/materials-editor.ui
-)
-
-add_library(MatGui SHARED ${MatGui_SRCS} ${MatGuiIcon_SVG} ${MatGuiImages} ${Material_Ui_Files})
+add_library(MatGui SHARED ${MatGui_SRCS} ${MatGuiIcon_SVG} ${MatGuiImages})
```

Comment:
I think you'll find that the file is missing in the built version. It will work fine in your repository build but won't be included in the installer. That's the purpose of this code